test/Serialization/type_wrapper_in_swiftinterface.swift

```diff
@@ -0,0 +1,93 @@
+// RUN: %empty-directory(%t/src)
+// RUN: %empty-directory(%t/sdk)
+// RUN: split-file %s %t/src
+
+/// Build the library.
+// RUN: %target-swift-frontend -emit-module %t/src/PublicModule.swift \
+// RUN:   -module-name PublicModule -swift-version 5 -enable-library-evolution \
+// RUN:   -emit-module-path %t/PublicModule.swiftmodule \
+// RUN:   -emit-module-interface-path %t/PublicModule.swiftinterface \
+// RUN:   -emit-private-module-interface-path %t/PublicModule.private.swiftinterface \
+// RUN:   -enable-experimental-feature TypeWrappers
```

Comment:
Many of the tests that enable `TypeWrappers` seems to include `REQUIRES: asserts`. This test does not, and if one runs the test suite in a non-assert compiler, one gets the error "experimental feature 'TypeWrappers' cannot be enabled in a production compiler". Would it be correct adding `REQUIRES: asserts` also to this test?